widget/ui/src/components/StatusDrawer/StatusDrawer.types.ts

```diff
@@ -0,0 +1,30 @@
+export interface PropTypes {
+  open: boolean;
+  onClose: () => void;
+  prefix?: React.ReactNode;
+  data: Data[] | Data;
+  footer?: React.ReactNode;
```

Comment:
I think you can remove the footer from the status drawer and get the footer and content of each status card from its props.